### ISSUE TITLE
Initial set of cipher suites

### ISSUE THREAD

**Issue by emanjon** [lake-wg/edhoc#120]
- Is it worth having 4 diiference CCM based ciphser suites. And are they the correct 4?

- EDHOC should probably have a ChaCha20-Poly1305 cipher suite from start. Probably with SHA-256. X25519 and EdDSA.

- The CNSA cipher suite does not really need a 1 byte value. Could live with 2 byte value.

**Comment by emanjon** [comment]
-Agreement during interim is that the 4 different CCM are all needed and that they are the correct ones.
-ChaCha20 should be combined with both P-256 and X25519
-CNSA suite can have two bytes.

**Comment by emanjon** [comment]
I think we should specify a X448/SHAKE256 suite to complement the CNSA suite.

**Comment by emanjon** [comment]
The GCM suite is chosen to match what is currently used on the web which is X25519 combined with P-256 ECDSA. Not sure we need to specify anything else here. IoT community seems more interested in ChaCha20 than GCM.

~~~~~~~~~~~
   6. ( 1, -16, 4, -7, 1, -16 )
      (A128GCM, SHA-256, X25519, ES256,
       A128GCM, SHA-256)
~~~~~~~~~~~

**Comment by emanjon** [comment]
https://github.com/lake-wg/edhoc/pull/127

**Comment by gselander** [comment]
Resolved  with #127.
